packages/engineer/src/feature/gui.dev-server.env.ts

```diff
@@ -14,7 +14,7 @@ guiFeature.setup(
         {
             buildFeature: {
                 engineerWebpackConfigs,
-                devServerConfig: { title, publicConfigsRoute },
+                devServerConfig: { title, favicon, publicConfigsRoute },
```

Comment:
why do we need it here?
this is the configuration for the engineer dashboard

Reply:
added it next to places that has `title`.
So what's the usage of `title` overthere?

Reply:
well.. not sure, dont see any good reason.

it's fine by me to keep it like this